### DISCUSSION TITLE
sync account with blockchain after get error

### DISCUSSION THREAD

**Discussion by dualsystems** [informalsystems/hermes#1646] [Q&A] [answered]
may be in this point refresh account data?

https://github.com/informalsystems/ibc-rs/blob/45ac6cac0ebc9f013b1892b018102e7fd19fdb8b/relayer/src/chain/cosmos.rs#L361

**Comment by adizere** [comment, marked as answer]
Thanks for your suggestion. The refresh approach is handled in this PR https://github.com/informalsystems/ibc-rs/pull/1349.
